services/libs/aster-std/src/process/process/mod.rs

```diff
@@ -71,6 +72,9 @@ pub struct Process {
     umask: Arc<RwLock<FileCreationMask>>,
     /// resource limits
     resource_limits: Mutex<ResourceLimits>,
+    /// Scheduling priority
+    /// According to POSIX, the nice value is a per-process setting.
+    nice: Mutex<Nice>,
```

Comment:
Using `Mutex` looks to too heavy-weight. I can't think of any use cases where the nice value must be accessed within a critical section. How about wrapping an `AtomicU8` inside `Nice`?

Reply:
Yes, using `Mutex` seems too heavy, I changed it to AtomicU8 inside `Nice`.